.gitignore

```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
```

Comment:
Do we need to commit changes to this .gitignore?

Reply:
I added because I built it locally, but I can remove it.